src/Control/Monad/Bayes/Population.hs

```diff
@@ -274,3 +276,7 @@ hoist ::
   Population m a ->
   Population n a
 hoist f = fromWeightedList . f . population
+
+-- | Normalizes the weights in the population so that their sum is 1.
+normalize :: (Monad m) => Population m a -> Population m a
+normalize = hoist unweighted . extractEvidence
```

Comment:
`unweighted` used to be called `prior` https://hackage.haskell.org/package/monad-bayes-0.1.1.0/docs/src/Control.Monad.Bayes.Population.html#normalize, so I guess this is the same implementation as before. Shouldn't we restore the comment about bias here?